src/App.tsx

```diff
@@ -17,18 +61,32 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                hasFilterChange={setSelectFilter}
```

Comment:
The prop `hasFilterChange` should be renamed to `onFilterChange` to better indicate that it is a callback function triggered when the filter changes.